contrib/IECoreImage/python/IECoreImage/__init__.py

```diff
@@ -0,0 +1,64 @@
+##########################################################################
+#
+#  Copyright (c) 2017, Ivan Imanishi. All rights reserved.
```

Comment:
Haven't added license/copyright headers before.
I've set it to my own name and the current year after copying it from another module.
Is that how it's supposed to be done?
Or should I leave it with your name @johnhaddon?
And I've also seen some with a year range instead of a single year.

Reply:
If you're doing it as an Image Engine employee, it'll be `(c) 2017, Image Engine Design Inc.`. The date should be the year of publishing. You can update it to be a range if you make edits in subsequent years, although I've heard people who sound like they know about these things say that keeping the date updated doesn't really matter.

Reply:
It does have the Image Engine line there. Do you mean that I should just remove the line with my name and keep the one for Image Engine?